.github/workflows/publish-maven.yml

```diff
@@ -0,0 +1,25 @@
+name: Publish Maven package
+on: [workflow_call, workflow_dispatch]
+
+jobs:
+  publish-maven:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+      packages: write
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+      - name: Set up JDK 14
+        uses: actions/setup-java@v1
+        with:
+          java-version: 14
+      - name: Publish maven-jar package
+        run: cd ${{ github.workspace }}/maven-jar && mvn --batch-mode deploy
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Publish maven-service package
```

Comment:
What are your thoughts on individual files... so allow for easier copy and paste....

so

* publish-maven-jar.yml
* publish-maven-service.yml